htdocs/content/buildings.php

```diff
@@ -572,7 +572,7 @@
 
 				while ($tarr = mysql_fetch_array($tres))
 				{
-					tableStart($tarr['type_name'],TABLE_WIDTH);
+					tableStart($tarr['type_name'],TABLE_WIDTH,"","","compact");
```

Comment:
Should this only be for `$cu->properties->itemShow!='full'`? Same as with defense?

Reply:
Fixed in new commit